modules/nextflow/src/main/groovy/nextflow/cli/CmdConfig.groovy

```diff
@@ -50,6 +50,9 @@ class CmdConfig extends CmdBase {
     @Parameter(names=['-profile'], description = 'Choose a configuration profile')
     String profile
 
+    @Parameter(names = '-property', description = 'Print the value of a config property, or fail if the property is not defined.')
```

Comment:
I was thinking if it should not be used something more idiomatic, e.g `-value` or `-eval`? thoughts ?

Reply:
Also makes sense. My thinking was that `-property` is most appropriate because there is already a `-properties` option.